### PR TITLE
fix(bug) fixing chat

### PR DESCRIPTION
**Pull request by edwinnambaje**
#### What does this PR do?
- Fix chat

#### Description of Task to be completed?
- Fix chat

#### How should this be manually tested?
after cloning the repo
1. `npm i` to install all pacheges need
2. `npm start` to start the project 
3. you can follow the documentation on path `/api-docs`

### CODE REVIEW

**Review by blackd44 — APPROVED** [review]

**Review by Zeden12 — APPROVED** [review]

File: src/helpers/socket.js [hunk 1]
```diff
@@ -1,94 +1,125 @@
...
+// import socketio from 'socket.io';
+// import { findById } from '../services/user.services';
+// import { Jwt } from './jwt';
+// import { addMessage, getMessages } from '../services/chat.services';
+// import { formatMessage } from './chat/message';
+// import {
+//   userJoin,
+//   getCurrentUser,
+//   userLeave,
+//   getRoomUsers
```
Comment (UmuhireJessie) [inline src/helpers/socket.js]: What are all these comments for? Explain the use of these here

File: src/services/chat.services.js [hunk 2]
```diff
@@ -3,28 +3,68 @@
...
+//   try {
+//     return Chat.findAll({
+//       where: { room },
+//       include: [
+//         {
+//           model: users,
+//           as: 'user',
+//           attributes: ['username']
+//         }
+//       ],
```
Comment (UmuhireJessie) [inline src/services/chat.services.js]: Why all these?

File: tests/chat.test.js [hunk 3]
```diff
@@ -31,17 +31,18 @@ describe('Chatting functionalities', () => {
...
+    //     .post('/chat/message')
+    //     .send({ message: 'Hello, world!' })
+    //     .set('Authorization', `Bearer ${sellerToken}`)
+    //     .end((err, res) => {
+    //       console.log(res.body);
+    //       expect(res).to.have.status(200);
+    //       expect(res.body.message).to.equal('Message sent.');
+    //       done();
+    //     });
+    // });
```
Comment (UmuhireJessie) [inline tests/chat.test.js]: Clean up this file

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Please see comments!

**Review by Patienceineza — APPROVED** [review]
LGTM

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**